server/config/config.go

```diff
@@ -774,6 +774,8 @@ type ScheduleConfig struct {
 	// EnableDiagnostic is the the option to enable using diagnostic
 	EnableDiagnostic bool `toml:"enable-diagnostic" json:"enable-diagnostic,string"`
 
+	// OperatorTimeoutOffset is the offset of operator timeout, default is 0.
+	OperatorTimeoutOffset typeutil.Duration `toml:"operator-timeout-offset" json:"operator-timeout-offset"`
```

Comment:
It's a little bit hard to understand the purpose through the name.

Reply:
yes, I think this configuration should not used in normal cluster.